### PR TITLE
add feed partial template + less file

### PR DESCRIPTION
**Pull request by ginold**
related to #754

@asaunier, when you integrate this into the preferences/user profile, you can use 
`<app-feed app-feed-profile="false" class="feed" ></app-feed>` 
anywhere you want. 

Then, don't forget to add the parent class into the `feed.less` file on the top. You may still want to change the bg color or the width of the feed in the less file. 

For example, just add `.preferences` next to `.home-page`, but then if you want to have a different color of the feed bg, type at the bottom of the file a new line `.preferences .feed { bg-color }` (!important will maybe not be needed)

### CODE REVIEW

**Comment by asaunier** [comment]
Do you really need ``class="feed"`` in ``<app-feed app-feed-profile="false" class="feed" ></app-feed>``?
Can't you use ``app-feed`` as the CSS selector instead?

> Then, don't forget to add the parent class into the feed.less file on the top.

Is it really necessary to add the parent class? Why can't I simply plug the directive in the user profile page and have it displayed out of the box?

**Comment by ginold** [comment]
> Do you really need class="feed"

not really, i could write in css app-feed, but as it turns out, the good practices of css tell to always have a class instead of a general element call. (better to style .feed than app-feed)

>  Why can't I simply plug the directive in the user profile page and have it displayed out of the box?

i can simplify and it will be 'plugable' out ouf the box, but you will still surely have some adaptations to do.

**Comment by ginold** [comment]
updated and ready to be plugged-in outta box!

File: c2corg_ui/templates/index.html [hunk 1]
```diff
@@ -37,53 +37,5 @@ <h1 class="welcome-title text-center" translate>The mountain sports community</h
...
-          <div class="list-item">
-            <app-card app-card-doc="doc['document']" class="{{type}}"></app-card>
-          </div>
-        </article>
-        <h4 class="text-success text-center" translate ng-show="feedCtrl.busy">Loading feed, please wait...</h4>
-      </div>
-
-    </div>
-  </div>
+  <app-feed app-feed-profile="false" id="feed"></app-feed>
```
Comment (asaunier) [inline c2corg_ui/templates/index.html]: I don't think you need to explicitly set ``app-feed-profile="false"``. ``false`` is the default value I think.

Comment (ginold) [inline c2corg_ui/templates/index.html]: seems to work :+1:

**Pull request closed without merging**